consensus/src/main/java/org/apache/iotdb/consensus/multileader/MultiLeaderConsensus.java

```diff
@@ -308,6 +308,11 @@ public ConsensusGenericResponse removePeer(ConsensusGroupId groupId, Peer peer)
     return ConsensusGenericResponse.newBuilder().setSuccess(true).build();
   }
 
+  @Override
+  public ConsensusGenericResponse updatePeer(ConsensusGroupId groupId, Peer peer) {
+    return ConsensusGenericResponse.newBuilder().setSuccess(true).build();
+  }
+
   @Override
```

Comment:
What is the exact semantic of `updatePeer`? 
What does `peer` param mean, it is the new peer? Should we also include original old peer?  
When updatePeer returns, should consensus guarantee the new peer can directly accept requests?
When updatePeer fails, what's the rollback strategy?
We should have a more thorough discussion and provide a document.